CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Fixes
+
+- Add debug_meta to all events ([#1756](https://github.com/getsentry/sentry-dart/pull/1756))
```

Comment:
Can we add another sub-item with: Fixes obfuscated stacktraces occurring on captureMessage and captureEvent calls when `attachStacktrace`option is used. 

or similar